src/main/scala/viper/gobra/frontend/Desugar.scala

```diff
@@ -243,6 +244,38 @@ object Desugar {
       in.MPredicateProxy(id.name, name)(meta(id, context))
     }
 
+    def functionLitProxyD(lit: PFunctionLit, context: TypeInfo): in.FunctionLitProxy = {
```

Comment:
did you consider using `in.FunctionProxy` instead of introducing `in.FunctionLitProxy`? If so, why did you decide against it?